.travis.yml

```diff
@@ -19,7 +19,7 @@ matrix:
       dist: trusty
```

Comment:
I suggest removing the HHVM job (as they don't care about PHP compat anymore, I don't care about knowing whether the lib works on HHVM or no)

And we could add the job testing with dev versions of deps